Easydict/Swift/View/AdvancedTabStyle.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  AdvancedTabStyle.swift
+//  Easydict
+//
+//  Created by Jerry on 2024-05-06.
+//  Copyright © 2024 izual. All rights reserved.
+//
+
+import SwiftUI
+
+/// Takes in a Color, a systemImage, a text label, and an optional subtitle to quickly create a toggle or picker style for Advanced Tab in Settings.
+struct AdvancedTabStyle: View {
```

Comment:
The name is a bit confusing, maybe something like `AdvancedTabItemView`.

Reply:
Thank you so much for pointing these out